target/checkstyle-cachefile

```diff
@@ -0,0 +1,3 @@
+#Fri May 27 16:04:05 EDT 2022
+configuration*?=19E3BA274DAE61391DB87338ADE724EFCE5FA334
```

Comment:
Is this a sensitive ID?

Reply:
Thanks @aforward . I'm not sure if it's sensitive or not (generated by `mvn install` command) but I did not mean to include it. I've removed it from this branch and added the requisite `.gitignore` file.